File: siem/service/src/main/java/bsep/sw/hateoas/log/LogAttributes.java
package bsep.sw.hateoas.log;

import org.joda.time.DateTime;

public class LogAttributes {

    public String level;
    public DateTime time;
    public LogInfoHateoas info;
    public String message;

    public LogAttributes level(final String level) {
        this.level = level;
        return this;
    }

    public LogAttributes time(final DateTime time) {
        this.time = time;
        return this;
    }

    public LogAttributes info(final LogInfoHateoas info) {
        this.info = info;
        return this;
    }

    public LogAttributes message(final String message) {
        this.message = message;
        return this;
    }

}

File: agent-win/WinAgent/model/LogAttributes.cs
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WinAgent.model
{
    class LogAttributes
    {
        [JsonConverter(typeof(StringEnumConverter))]
        public LogLevel Level { get; set; }
        public string Time { get; set; }
        public LogInfo Info { get; set; }
        public string Message { get; set; }

        public LogAttributes() { }

        public LogAttributes(Log log)
        {
            this.Message = log.Message;
            this.Level = log.Level;
            this.Info = new LogInfo(log);
            this.Time = log.Time.ToString("yyyy-MM-dd'T'HH:mm:ssZ");
        }

    }
}
